tests/unit/jobserver/api/test_jobs.py

```diff
@@ -442,6 +442,37 @@ def test_jobapiupdate_post_only(api_rf):
     assert JobAPIUpdate.as_view()(request).status_code == 405
 
 
+@pytest.mark.django_db
+def test_jobapiupdate_post_with_internal_error(api_rf, mocker):
+    backend = BackendFactory()
+    job_request = JobRequestFactory()
+
+    now = timezone.now()
+
+    mocker.patch("jobserver.api.jobs.sentry_sdk", autospec=True)
+
+    data = [
+        {
+            "identifier": "job",
+            "job_request_id": job_request.identifier,
+            "action": "test-action",
+            "status": "running",
+            "status_code": "",
+            "status_message": "Internal error",
+            "created_at": now,
+            "started_at": now,
+            "updated_at": now,
+            "completed_at": None,
+        },
+    ]
+    request = api_rf.post(
+        "/", HTTP_AUTHORIZATION=backend.auth_token, data=data, format="json"
+    )
+    response = JobAPIUpdate.as_view()(request)
+
+    assert response.status_code == 200, response.data
```

Comment:
Is it possible to check that that we send the error to Sentry?

Reply:
Good catch, I've added an assert to check we're calling `capture_message()`.  It doesn't look like sentry ships any of their testing tools with the SDK for checking you've called it correctly unfortunately, would be nicer than mocking.